release/_releases_json.sh

```diff
@@ -39,6 +39,12 @@ function _generate_product_version_list {
 	echo "${directory_html}"
 }
 
+function _get_latest_product_version {
+	echo "$(_generate_product_version_list "${1}")" | \
+		grep -oP '(?<=<a href=")(?:\d{4}\.q[1-4]\.\d+(?:-lts)?|7\.4\.3\.\d+-ga\d+)(?=/")' | \
```

Comment:
@davisaints, can you please use the full name of the `grep` parameters? For example: `--only-matching` and `--perl-regexp`.

In addition, can you please include 7.3 release in this regex?